testing/jwts.go

```diff
@@ -16,9 +16,10 @@ var (
 func NewToken(body *JwtBody) string {
 	logging.RootLog.Debug("Creating JWT with body: %v", body)
 	claims := jwt.MapClaims{
-		"sub":   body.Subject,
-		"roles": body.Roles,
-		"iss":   body.Issuer,
+		"sub":     body.Subject,
+		"user_id": body.UserId,
```

Comment:
instead of the "name" string, pass in a `map[string]string` and then process it adding claims; this is already supported by the framework anyway.

Reply:
Updated to 
```golang
	claims := jwt.MapClaims{}
	for k, v := range body.Claims {
		claims[k] = v
	}
```

Reply:
no, that's not at all what I meant
`sub` and `roles` are "native" elements of the JWT - you would only use `claims` (in the Go struct) to add other "claims" (still as `"k": "v"` in the JWT's JSON)